### DISCUSSION TITLE
About ControlNet train with the 1step PixArt-DMD in diffusers.

### DISCUSSION THREAD

**Discussion by CuddleSabe** [PixArt-alpha/PixArt-alpha#134] [Q&A]
Hi, I found that the 1step model PixArt-DMD is supported to the diffusers. So I write a train script in diffusers,
the output is chunk to two components: pred_noise and vb,
I only use pred_noise to compute mse loss, the loss decrease very fast, but the output image is very bad
what should I do?

<img width="1359" alt="截屏2024-04-28 12 26 57" src="https://github.com/PixArt-alpha/PixArt-alpha/assets/61224076/9f63cd2d-f703-43f2-9801-81fba06fcb6a">

**Comment by CuddleSabe** [comment]
add up to 256 batch size with 8 a100.

**Comment by lawrence-cj** [comment]
We released here: https://github.com/PixArt-alpha/PixArt-sigma/blob/master/asset/docs/pixart_dmd.md

**Reply by CuddleSabe** [reply]
hi, I found when I "with torch.autocast("cuda"):" for the diffusers PixArt pipeline,
![validation_0_fe0c3de4f9a024ad739f](https://github.com/PixArt-alpha/PixArt-alpha/assets/61224076/f770fbf2-2d61-41d6-8c05-e3822117a97a)
the output will like this
